sky/task.py

```diff
@@ -118,6 +118,7 @@ def __init__(
         # Advanced:
         docker_image: Optional[str] = None,
         event_callback: Optional[str] = None,
+        blocked_resources: Optional['resources_lib.Resources'] = None,
```

Comment:
Add to Args? Should it be an Optional[Set/Iterable[...]] ?

Reply:
Good point! Added. Thanks!